scripts/macho2segments.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+'use strict';
+
+const assert = require('assert');
+const fs = require('fs');
+const Buffer = require('buffer').Buffer;
```

Comment:
That's how I always roll :) Maybe one day we will be able to remove global `Buffer`.